File: CodeForces/CF_1941A.cpp
#include <bits/stdc++.h>

using namespace std;
using ll = long long;

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    int T;
    cin >> T;
    while (T--)
    {
        ll n,m,k;
        cin>>n>>m>>k;
        vector<ll> b(n);
        vector<ll> c(m);
        for(int i = 0; i< n; ++i){
            cin>>b[i];
        }
        for(int i = 0; i<m;++i){
            cin>>c[i];
        }
        ll count = 0;
        for(int i = 0; i< n; ++i){
            for(int j = 0; j<m; ++j){
                if(b[i] + c[j] <= k){
                    count += 1; 
                }
            }
        }
        cout<<count<<"\n";
    }
    return 0;
}

File: CodeForces/CF_1941A.py
T = int(input())

while(T):
    T-=1
    n, m, k = map(int, input().split())
    b = list(map(int, input().split()))
    c = list(map(int, input().split()))

    count = 0
    for x in b:
        for y in c:
            if(x + y <=k):
                count += 1

    print(count)

